### PR TITLE
Translating SRF Home to python

### PR DESCRIPTION
**Pull request by lisazacarias**
The primary two changes of the srfhome python conversion are:
1. watcher creation is streamlined so that all watchers are launched in the same way (no shell scripts needed)
2. shortcut buttons are now for the most part automatically added from appropriately named files in the repo. A handful are manually added because they refer to files in production

### CODE REVIEW

**Review by jjanice — APPROVED** [review]
Yay! Only one comment on a comment :)

File: displays/srfhome/srf_home.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            QSizePolicy.Preferred, QSizePolicy.Maximum
+        )
+        self.fill_mini_home_groupbox()
+
+        self.setLayout(self.main_layout)
+
+        self.links_groupbox = QGroupBox("Shortcuts && Bookmarks")
+        self.fill_link_groupbox()
+
+        sel_opt_path = (
```
[the remark this answers is not in the document]
Comment (lisazacarias) [reply displays/srfhome/srf_home.py:41]: Oh yeah I think I thought of that and forgot lol

File: displays/srfhome/utils.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    groupbox = QGroupBox(f"{parsed_name} Watcher")
+    vlayout = QVBoxLayout()
+    groupbox.setLayout(vlayout)
+
+    show_button = PyDMShellCommand()
+    show_button.setText(f"Show {parsed_name} Output")
+    xterm_prefix = f"xterm -T {watcher_name} -hold -e"
+    show_button.commands = [f"{xterm_prefix} tmux_launcher open {watcher_name}"]
+
+    xterm_prefix += ' "export TMUX_SSH_USER=laci && export TMUX_SSH_SERVER=lcls-srv03'
```
[the remark this answers is not in the document]
Comment (lisazacarias) [reply displays/srfhome/utils.py:17]: It should get matched in the full commands under it (I hope)

[the remark this answers is not in the document]
Comment (lisazacarias) [reply displays/srfhome/utils.py:17]: Ya in the two tumux commands

[the remark this answers is not in the document]
Comment (hmarts9) [reply displays/srfhome/utils.py:17]: Oh gotcha ok!

**Pull request merged**